docs/src/pages/customization/density/density.md

```diff
@@ -17,19 +17,19 @@ reducing the size.
 
 The following components have props applying higher density:
 
-- [Button](/api/button)
-- [Fab](/api/fab)
-- [FilledInput](/api/filled-input)
-- [FormControl](/api/form-control)
-- [FormHelperText](/api/form-helper-text)
-- [IconButton](/api/icon-button)
-- [InputBase](/api/input-base)
-- [InputLabel](/api/input-label)
-- [ListItem](/api/list-item)
-- [OutlinedInput](/api/outlined-input)
-- [Table](/api/table)
-- [TextField](/api/text-field)
-- [Toolbar](/api/toolbar)
+- [Button](/api/button/)
```

Comment:
Would it make sense to lint for these issues? I see this change fairly often. Best with a rationale included which I'm still missing: What is the difference between trailing slash and not? From a server perspective the trailing slash isn't accurate because they don't live in api/button/index.js but api/button.js.

Reply:
You get the underlying subfolder/index.html vs subfolder.html difference right. It seems to be a legacy issue. Next.js used to add a leading slash automatically, with no way to change it, until v9: https://github.com/zeit/next.js/issues/2944.

Reply:
So the issue is that it creates a 301 without the trailing slash? 

If the trailing slash has no other implications we might as well use the default behavior instead.

Reply:
Yes, the issue is the 301 we have right now in production.

What default behavior do you refer to?

Reply:
> What default behavior do you refer to?

The one from next adding trailing slashes to every link.

Reply:
Next doesn't handle markdown.

Reply:
As we are using a custom express server now. We would make the dev env have trailing slashs, to behave like production.